database/memory/memory.go

```diff
@@ -0,0 +1,84 @@
+package memory
+
+import (
+	"errors"
+	"soarca/models/api"
+	"soarca/models/cacao"
+	"soarca/models/decoder"
+)
+
+type InMemoryDatabase struct {
+	playbooks map[string]cacao.Playbook
+}
+
+func New() InMemoryDatabase {
```

Comment:
should we rename this? this seems the IplaybookRepositority interface that you are implementing, rather than the database interface. What are your thoughts?